tests/smoke_test.py

```diff
@@ -6,11 +6,18 @@
 
 
 if __name__ == "__main__":
+    # ASM IAST smoke test
+    if sys.version_info >= (3, 6, 0) and system() != "Windows":
+        from ddtrace.appsec.iast._taint_tracking._native import ops
+
+        assert ops
```

Comment:
Should we log a message when this assertion fails? Is it sufficient to check if ops is a truthy value? Should we add any additional validations?

Reply:
Yes, if IAST is enabled but the SO isn't supported IAST could raise a ImportError